src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Mod_Float.txt

```diff
@@ -32,4 +32,4 @@
 -5.8180083079168396E-201
 
 >> Mod(1E+400, 3)
-Errors: Error 4-10: Numeric value is too large.|Error 0-14: The function 'Mod' has some invalid arguments.|Error 4-10: Invalid argument type (Error). Expecting a Number value instead.
```

Comment:
With the more fleixble comparsion in baserunner.cs, are these changes needed now?

Reply:
not need, just for testing that even message return are different, we still can pass the check.

Reply:
Maybe i just modify Mod test and no change for text_format to test this change here.